bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqBinding.java

```diff
@@ -0,0 +1,369 @@
+﻿/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.myq.internal;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.openhab.binding.myq.MyqBindingProvider;
+import org.openhab.binding.myq.internal.MyqBindingConfig;
+import org.openhab.binding.myq.internal.GarageDoorDevice.GarageDoorStatus;
+import org.openhab.binding.myq.internal.MyqData.InvalidLoginException;
+import org.apache.commons.lang.StringUtils;
+import org.openhab.core.binding.AbstractBinding;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.PercentType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.library.types.OpenClosedType;
+import org.openhab.core.library.types.UpDownType;
+import org.openhab.core.types.Command;
+import org.osgi.framework.BundleContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * this class polls the Chamberlain MyQ API and sends updates to the event bus
+ * of configured items in openHAB
+ * 
+ * @author Scott Hanson
+ * @author Dan Cunningham
+ * @since 1.8.0
+ */
+public class MyqBinding extends AbstractBinding<MyqBindingProvider> {
+	private static final Logger logger = LoggerFactory
+			.getLogger(MyqBinding.class);
```

Comment:
Please the openHAB Eclipse formatter over all Java files (120 max line length, etc.)